docs/getting-started/navigating-the-toolkit.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Navigating the Toolkit
+description: Getting to know the Powertools for AWS Lambda Toolkit
+---
+
+<!-- markdownlint-disable MD043 -->
+
+Powertools for AWS Lambda (TypeScript) is a collection of utilities designed to help you build serverless applications on AWS.
+
+The toolkit is designed to be modular, so you can pick and choose the utilities you need for your application. Each utility is designed to be used independently, but they can also be used together to provide a complete solution for your serverless applications.
+
+## Patterns
+
+Many of the utilities provided can be used with different patterns, depending on your preferences and the structure of your code.
+
+### Class Method Decorator
+
+If you prefer writing your business logic using Object-Oriented Programming (OOP) and TypeScript Classes, the Class Method decorator pattern is a good fit. This approach lets you decorate class methods with Powertools utilities, applying their functionality with minimal code changes.
+
+This pattern works well when you want to integrate Powertools for AWS Lambda (TypeScript) into an existing codebase without significant refactoring and with no additional runtime dependencies. Note that this approach relies on TypeScript's experimental decorator feature.
+
+```ts
```

Comment:
Should we extract this example into a snippet as we do with other lambda function examples?